internal/config/config.go

```diff
@@ -5,23 +5,32 @@ import (
 	"os"
 )
 
-var (
-	Server  *string
-	BaseURL *string
-)
+type Config struct {
+	server  *string
+	baseURL *string
+}
+
+var globalConfig = Config{
+	server:  nil,
+	baseURL: nil,
+}
+
+func GetBaseURL() *string {
+	return globalConfig.baseURL
+}
 
 func Init() string {
-	Server = flag.String("a", ":8080", "Server URL")
-	BaseURL = flag.String("b", "http://localhost:8080", "Base URL")
+	globalConfig.server = flag.String("a", ":8080", "Server URL")
+	globalConfig.baseURL = flag.String("b", "http://localhost:8080", "Base URL")
 
 	flag.Parse()
 
-	if envServerAddress := os.Getenv("SERVER_ADDRESS"); envServerAddress != "" {
-		Server = &envServerAddress
+	if envServerAddress, exists := os.LookupEnv("SERVER_ADDRESS"); exists {
```

Comment:
Если функция в Go возвращает вторую переменную булевого типа, которая указывает на корректность выполнения операции и т.п., ее принято называть `ok`.

```suggestion
	if envServerAddress, ok := os.LookupEnv("SERVER_ADDRESS"); exists {
```